sites/next.skeleton.dev/src/content/docs/components/segment-control/svelte.mdx

```diff
@@ -88,3 +87,8 @@ Consider using a Skeleton [Button Group](/docs/tailwind/buttons#group) if you ne
 		<Code code={ExampleGroupRaw} lang="svelte" />
 	</Fragment>
 </Preview>
+
+## API Reference
+
+<ApiTable types={['SegmentProps', 'SegmentItemsProps']} schema={SegmentSchema} />
```

Comment:
This is not displaying on the page for some reason.

Reply:
Fixed, there was a name mismatch between the requested types and the types present. I've actually renamed the types in the skeleton package as it was out of order with the rest of the package `SegmentControl` -> `SegmentProps`.